aten/src/ATen/native/nested/NestedTensorBackward.cpp

```diff
@@ -13,7 +13,12 @@
 namespace at {
 namespace native {
 
-// See Note [nested tensor matmul] TODO in NestedTensorMath.cpp
+Tensor native_dropout_backward_nested(const Tensor& grad, const Tensor& mask, double scale) {
+  Tensor result = grad * mask * scale;
+  return result;
+}
```

Comment:
Do we even need this since it's the same as the CPU implementation?

Reply:
I think 1, maybe rename `native_dropout_backward_cpu` to something a little bit more neutral. I don't see us needing future updates, and I think the ideal state is no new logic needed for NT whenever possible. If we need to split them later on for some reason, best to do it then imo.